python_src/src/lamp_py/tableau/pipeline.py

```diff
@@ -0,0 +1,35 @@
+from lamp_py.tableau.jobs.rt_rail import HyperRtRail
+from lamp_py.tableau.jobs.gtfs_rail import (
+    HyperServiceIdByRoute,
+    HyperStaticCalendar,
+    HyperStaticCalendarDates,
+    HyperStaticFeedInfo,
+    HyperStaticRoutes,
+    HyperStaticStops,
+    HyperStaticStopTimes,
+    HyperStaticTrips,
+)
+
+HYPER_JOBS = (
+    HyperRtRail(),
+    HyperServiceIdByRoute(),
+    HyperStaticCalendar(),
+    HyperStaticCalendarDates(),
+    HyperStaticFeedInfo(),
+    HyperStaticRoutes(),
+    HyperStaticStops(),
+    HyperStaticStopTimes(),
+    HyperStaticTrips(),
+)
+
+
+def start_hyper_updates() -> None:
+    """Run all HyperFile Update Jobs"""
+    for job in HYPER_JOBS:
+        job.run_hyper()
```

Comment:
we should run a `validate_environment` function here.

it logs all of the env vars we care about and makes debugging easier.

Reply:
Incorporated